include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -184,9 +234,9 @@ SECTIONS
 
     GROUP_START(RAMABLE_REGION)
 
-#if defined(CONFIG_SOC_SERIES_STM32F0X) && !defined(CONFIG_IS_BOOTLOADER)
+#if defined(CONFIG_CPU_CORTEX_M0) && (defined(CONFIG_IS_BOOTLOADER) || defined(CONFIG_IMG_MANAGER))
     /* Must be first in ramable region */
-	SECTION_PROLOGUE(.st_stm32f0x_vt,(NOLOAD),)
+	SECTION_PROLOGUE(.ram_vector_table,(NOLOAD),)
 	{
         _ram_vector_start = .;
```

Comment:
Here @nvlsianpu , application and bootloader both need to reserve this memory array. When application is chainloaded, the prep_C() will put the vector table here.